### PR TITLE
replace cypress uses of waitFor with wait

### PR DESCRIPTION
**Pull request by duncan-truss**
## Summary

While reviewing Cypress work around PPMs for A-team, we discovered that there is some mixup with using waitFor() instead of wait() for waiting on api request intercepts to finish loading.  I know in our jest tests it's common to call waitFor() for asynchronous behavior so that can be part of the confusion.

The waitFor method in Cypress is about waiting for javascript events to be called https://github.com/EventEmitter2/EventEmitter2#emitterwaitforevent--eventns-options

Someone else has encountered this weirdness being exposed in Cypress as well https://github.com/cypress-io/cypress/issues/20556

I looked and this doesn't seem to be the root cause of any of our flaky e2e test from what I saw but it's good to stamp out this pattern since it silently just doesn't do what is intended.  You have to look at the cypress output to see that the wait statement isn't displayed like the others.

## Setup to Run Your Code

💻 You will need to use one terminal to test this locally

##### Terminal 1

Start the Cypress interactive tests locally.

```sh
make e2e_test_fresh
```

</details>

### Additional steps

1. Run all the services counseling Cypress flows
2. Run the TOO flows for NTS and NTSR
3. Run the homepage flow

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

File: cypress/integration/office/servicescounseling/servicesCounselingNTS.js [hunk 1]
```diff
@@ -50,7 +51,7 @@ describe('Services counselor user', () => {
     navigateToMove('NTSHHG');
 
     cy.get('[data-testid="ShipmentContainer"] .usa-button').last().click();
-    cy.waitFor(['@getMTOServiceItems']);
+    cy.wait(['@getMTOShipments']);
```
Comment (felipe-lee) [inline cypress/integration/office/servicescounseling/servicesCounselingNTS.js:54]: Should the ones that were changed from waiting for `'@getMTOServiceItems'` to `'@getMTOShipments'` be changed back? Looking at the query (at least I think I'm looking at the right one...), it looks like the service items is the last thing queried: 

https://github.com/transcom/mymove/blob/5c29f7a43d9c86d01a3bd6fba6d7ebaa9fc0f34e/src/hooks/queries.js#L361-L370

Comment (duncan-truss) [reply cypress/integration/office/servicescounseling/servicesCounselingNTS.js:54]: What you're linking to is the hook called when the Move Details page loads and that is being waited for here https://github.com/transcom/mymove/blob/df05c0a5d38ee1ad9137bcdf62d8f62cdeca38cf/cypress/integration/office/servicescounseling/servicesCounselingNTS.js#L116 inside of navigateToMove.

Clicking on that shipment container button is taking you to the edit shipment screen, which has a different hook useEditShipmentQueries https://github.com/transcom/mymove/blob/3e1cbfaccff6dc22f6080294aeb7613feee521d1/src/hooks/queries.js#L121

Comment (felipe-lee) [reply cypress/integration/office/servicescounseling/servicesCounselingNTS.js:54]: Ah, ok, thanks for pointing out the correct one!

**Review by felipe-lee — APPROVED** [review]
LGTM, thanks for fixing these!

**Pull request merged**